src/ui/studio.cpp

```diff
@@ -976,6 +979,124 @@ void CStudio::SetInfoText(std::string text, bool bClickable)
     }
 }
 
+// Beginning of the display of a dialogue.
+
+void CStudio::StartDialog(const Event &event)
+{
+    if ( event.type == EVENT_STUDIO_OPEN )
+    {
+        m_dialog = MakeUnique<CFileDialog>();
```

Comment:
I'd rather call this `m_fileDialog`, as `m_dialog` is used in other places referring to `CMainDialog`